crates/router/src/core/payment_link.rs

```diff
@@ -575,7 +590,8 @@ pub fn get_payment_link_config_based_on_priority(
         (
             enabled_saved_payment_method,
             DEFAULT_ENABLE_SAVED_PAYMENT_METHOD
-        )
+        ),
+        (merchant_details, IndexMap::new())
```

Comment:
Why are we sending `IndexMap::new()` i suppose we have defined a const for it in the consts file.